ui/src/common/components/ListWithPagination.jsx

```diff
@@ -70,8 +83,9 @@ ListWithPagination.propTypes = {
   pageItems: PropTypes.instanceOf(Immutable.List).isRequired,
   renderItem: PropTypes.func.isRequired,
   onPageChange: PropTypes.func.isRequired,
-  pageSize: PropTypes.number.isRequired,
-  page: PropTypes.number.isRequired,
+  onSizeChange: PropTypes.func,
```

Comment:
we need a test for this as well, and make sure `onSizeSize` is called when internal `Pagination.onShowSizeChange` triggered.